src/main/java/org/codinjutsu/tools/jenkins/view/BuildParamDialog.java

```diff
@@ -80,8 +80,9 @@ public static void showDialog(final Job job, final JenkinsAppSettings configurat
         SwingUtilities.invokeLater(new Runnable() {
             public void run() {
                 BuildParamDialog dialog = new BuildParamDialog(job, configuration, requestManager, runBuildCallback);
-                dialog.setLocationRelativeTo(null);
+                dialog.setSize(dialog.getPreferredSize());
                 dialog.setMaximumSize(new Dimension(300, 200));
+                dialog.setLocationRelativeTo(null);
```

Comment:
Is there any reason to move this line from line 83 to here?
If not, I would put it back where it was.

Reply:
I accidentally moved it, you could put it back.